# This patch makes sure the LD_LIBRARY_PATH is unset prior to running
# any lmod command. In this way, no matter what module is loaded,
# lmod will keep on working
# ToDo: csh
diff --git a/init/bash.in b/init/bash.in
index bdee611..7df2042 100644
--- a/init/bash.in
+++ b/init/bash.in
@@ -50,7 +50,7 @@ export MODULESHOME
 
 module()
 {
-  eval $($LMOD_CMD bash "$@")
+  eval $(LD_LIBRARY_PATH="" $LMOD_CMD bash "$@")
   [ $? = 0 ] && eval $(${LMOD_SETTARG_CMD:-:} -s sh)
 }
 
@@ -76,7 +76,7 @@ fi
 unalias ml 2> /dev/null || true
 ml()
 {
-  eval $($LMOD_DIR/ml_cmd "$@")
+  eval $(LD_LIBRARY_PATH="" $LMOD_DIR/ml_cmd "$@")
 }
 
 export_module=$(echo "@export_module@" | @tr@ '[:upper:]' '[:lower:]')
